zkevm-circuits/src/copy_circuit.rs

```diff
@@ -176,6 +187,8 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
         let [is_pad, is_tx_calldata, is_bytecode, is_memory, is_memory_copy, is_tx_log, is_access_list_address, is_access_list_storage_key] =
             array_init(|_| meta.advice_column());
         let is_first = copy_table.is_first;
+        #[cfg(feature = "dual_bytecode")]
+        let is_first_bytecode_table = meta.advice_column_in(SecondPhase);
```

Comment:
is the column boolean?  why need second phase?

Reply:
at earlier time considering to  use code_hash to verify it correctness,  changed the design later,  don't need phase now.  removed in  [121a5aa](https://github.com/scroll-tech/zkevm-circuits/pull/1368/commits/121a5aabcc900f92d7735e000844a16bfc506237)